.docksal/commands/php/ext

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+## Toggles a given PHP extension (most commonly xdebug).
+##
+## Usage: fin php/ext xhprof on|off
+#: exec_target = cli
+
+usage() {
+  echo "Usage:"
+  echo "fin php-ext <extension name> on|off"
+}
+
+toggle_on() {
+  sudo su -c "sed -ri 's/^[#;]((zend_)?extension)/\1/' ${EXTENSION_PATH}"
+  echo "${EXTENSION} on"
+}
+
+toggle_off() {
+  sudo su -c "sed -ri 's/^((zend_)?extension)/#\1/' ${EXTENSION_PATH}"
+  echo "${EXTENSION} off"
+}
+
+restart_php_fpm() {
+  sudo su -c "pkill --signal USR1 --pidfile /var/run/php-fpm.pid;"
```

Comment:
A more straightforward way, without causing error messages in the log from supervisor:

```suggestion
  sudo supervisorctl restart php-fpm
```